src/main/resources/application-worker.yaml

```
@@ -24,3 +24,9 @@ rhsm-subscriptions:
     keystore: file:${SUBSCRIPTION_KEYSTORE:}
     keystorePassword: changeit
     maxConnections: ${SUBSCRIPTION_MAX_CONNECTIONS:100}
+  product:
+    useStub: ${STUB_PRODUCT:true}
```

Two nitpicks: Let's default this to false instead, and rename env var to `PRODUCT_USE_STUB` for consistency (realizing the subscription product could use these changes too... :-/ appreciate if you want to go ahead and tackle that here too, but also happy to just make a tech debt card for it.
```suggestion
    useStub: ${PRODUCT_USE_STUB:false}
```

---

I **did** change this to be `PRODUCT_USE_STUB` as requested, but I **did not** switch this to `false`. When I did, various tests failed since a keystore file is also needed to be used, which this doesn't provide.

---

Ah, make sure that the test uses `@ActiveProfiles("test")` and then set `useStub` to `true` in `src/test/resources/application-test.yaml`